README.md

```diff
@@ -1,3 +1,42 @@
 # medic-builds-repo
 
 Design doc for builds repository.
+
+DDOCS are [pushed up by travis](https://github.com/medic/medic-webapp/blob/master/scripts/ci/push_to_staging.sh) with a specific name. These are then [pulled back down by api](https://github.com/medic/medic-api/blob/master/controllers/upgrade.js) and deployed by [horticulturalist](https://github.com/medic/horticulturalist).
+
+## Naming scheme
+
+Three valid examples: `medic:medic:3.0.0-beta.1`, `medic:medic:3.0.0`, `medic:medic:my-awesome-feature-branch`.
+
+The first two chunks between `:` is the namespace and the application name respectively. The namespace is just that: it has no logic relevant and is just used to scope things. The application name maps to the original ddoc name. It is what will be used to determine what to call the ddoc when deploying it.
```

Comment:
> and make this an argument 

What does this refer to here? If this refers to the db name, then we're not locking that down in this naming convention, you can (in theory) deploy to any db name you like. It's just hard-coded in api for now because I want to do one thing at time (see my comment on that PR).

If this refers to the ddoc name then, well, I don't know about other applications based in couchdb, but ours has our ddoc name hardcoded in a thousand different places, so you couldn't deploy that to another name even if you wanted to.